index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+    <title>My First Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <a class="welcome-message">Welcome</a>
+            <div>
+                <i class="ri-menu-line"></i>
+            </div>
+        </nav>
+    </header> 
+    <section class="Greetings">
+        <h1 id="intro1">Hey there. I'm Edgard</h1>
+        <h1 id="intro2">I'm a software developer</h1>
+        <p class="intromessage"> I can help you build a product , feature or website. 
+            Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.</p>
+    </section class="socialmedia">
```

Comment:
- [ ] Please remove this end of `section` since the section should end at line 34 and you already have its ending there.
- [ ] Kindly also note that element attributes can only be assigned in the opening tags and not closing tags.

Reply:
@NduatiKagiri I did not find any background image for template 3, just appear some vectors.

Reply:
@NduatiKagiri  I am having a hard time with the background images, should I just add each one of the vectors that appear in the template?

Reply:
@edgardsoza you only need to click on the image you want on the Figma design, then check the properties window on the right, and you will see three options on the top, Comment, Inspect, and Export. Click on the export part and will see options to choose the format you want to download the image in.